I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/r_A.rpy", line 54, in script
    show baba truecenter
Exception: Image 'baba' does not accept attributes 'truecenter'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/r_A.rpy", line 54, in script
    show baba truecenter
  File "C:\Users\shitt\Downloads\renpy-8.3.1-sdk\renpy\ast.py", line 1111, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Users\shitt\Downloads\renpy-8.3.1-sdk\renpy\ast.py", line 1074, in show_imspec
    renpy.config.show(name,
  File "C:\Users\shitt\Downloads\renpy-8.3.1-sdk\renpy\exports\displayexports.py", line 477, in show
    if not base.find_target() and renpy.config.missing_show:
  File "C:\Users\shitt\Downloads\renpy-8.3.1-sdk\renpy\display\image.py", line 421, in find_target
    self.target = target._duplicate(a)
  File "C:\Users\shitt\Downloads\renpy-8.3.1-sdk\renpy\display\displayable.py", line 249, in _duplicate
    args.extraneous()
  File "C:\Users\shitt\Downloads\renpy-8.3.1-sdk\renpy\display\displayable.py", line 109, in extraneous
    raise Exception("Image '{}' does not accept attributes '{}'.".format(
Exception: Image 'baba' does not accept attributes 'truecenter'.

Windows-10-10.0.22631 AMD64
Ren'Py 8.3.1.24090601
VisualNovel_Programming 1.0
Wed Jan 29 14:00:50 2025
